From df0afe4cfbdc135244a664b15babf03ef9417b9a Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 12 Mar 2017 13:57:38 +0200
Subject: [PATCH 0608/1051] dts: cp110: eip197: add cell index property

This patch adds cell-index property which allows to distinguish
between multiple engines in devices with 2 EIP197 engines.

Binding document was updated accordingly.

Change-Id: I5e7077d1af3b5cec117ec01c33c8136e5207daca
Signed-off-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-cp110.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
index b983be462d64..9f80b7ad8c1d 100644
--- a/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-cp110.dtsi
@@ -15,6 +15,7 @@
 #define CP110_PCIEx_REG0_BASE(iface)		(CP110_BASE + 0x600000 + (iface) * 0x20000)
 #define CP110_PCIEx_REG1_BASE(iface)		(CP110_PCIEx_CPU_MEM_BASE(iface) + CP110_PCIE_MEM_SIZE)
 
+#define CP110_EIP197_INDEX			CP110_NUM
 #define CP110_SPI_BUS_ID(n)			((CP110_NUM * 2) + (n))
 
 / {
@@ -432,6 +433,7 @@
 			clock-names = "core", "reg";
 			clocks = <&CP110_LABEL(clk) 1 26>,
 				 <&CP110_LABEL(clk) 1 17>;
+			cell-index = <CP110_EIP197_INDEX>;
 			dma-coherent;
 		};
 	};
-- 
2.17.1

